src/main/java/org/opentripplanner/raptor/rangeraptor/transit/AccessEgressFunctions.java

```diff
@@ -47,32 +52,64 @@ public final class AccessEgressFunctions {
    * </ol>
    */
   private static final ParetoComparator<RaptorAccessEgress> STANDARD_COMPARATOR = (l, r) ->
-    (l.stopReachedOnBoard() && !r.stopReachedOnBoard()) ||
-    r.hasOpeningHours() ||
-    (l.numberOfRides() < r.numberOfRides()) ||
-    (l.durationInSeconds() < r.durationInSeconds());
+    (
+      l.stopReachedOnBoard() &&
```

Comment:
Since this is pretty performance-critical stuff I have to ask: you only remove the brackets, isn't it?

Reply:
Yes, I only changed the brackets, but I see now that the unwanted line break is there, so I will fix that. FYI: This is not performance critical - only run before the raptor search to prune the access/egress. But, we want to keep it in the same style as the stop-arrival comparators which are critical.